aten/src/ATen/native/native_functions.yaml

```diff
@@ -2701,16 +2701,29 @@
 
 - func: matrix_exp_backward(Tensor self, Tensor grad) -> Tensor
 
+# DEPRECATED: Use torch.aminmax instead
 - func: _aminmax(Tensor self) -> (Tensor, Tensor)
   variants: function
   dispatch:
     CPU, CUDA: _aminmax_all
```

Comment:
This is probably fine to leave these dispatches in but technically these are composite implicit autograd now, right?

Reply:
I will go through a deprecation cycle with these dispatches just to be safe in case someone discovered these and started using them. And yes, they are composite implicit autograd though aminmax does not yet support autograd.